### ISSUE TITLE
GooleNet Training

### ISSUE THREAD

**Issue by dansileshi** [BVLC/caffe#2371]
![screenshot from 2015-04-27 01 27 36](https://cloud.githubusercontent.com/assets/8971279/7340120/d66c73bc-ec7d-11e4-9f91-8b4a2bc38ef9.png)
I am training my dataset using GoogleNet. How ever I have the following doubts.
1) The total loss is not the sum of the three losses (In this case in iteration 140100 the total loss is: .927989 is different form the the sum of the three losses .706834 +.272225 + .245504. But according to the GoogleNet paper the loss is the sum of the three losses i.e .3*loss1+ .3*loos2 + 1*loss3). Any suggestions? 
2) My training didn't improve more than around 71% since the 40,000th iteration now it is on 140,000 th
 iteration. When do I have to stop the training?

**Comment by seanbell** [comment]
1) The "loss" is the smoothed loss (40 iterations by default), but the individual outputs aren't smoothed.

2) If it's stopped improving, you should decrease the learning rate from 0.01.  You should consider using the "quick" solver file -- it has a more aggressive learning rate schedule that converges faster.

Also, you should really ask these kinds of questions on the [caffe user mailing list](https://groups.google.com/forum/#!forum/caffe-users).  Issues are for code discussions, not modelling discussions.

**Comment by shelhamer** [comment]
@seanbell is right on all counts -- closing (and thanks for answering).
